go.mod

```diff
@@ -109,5 +109,5 @@ replace (
 
 	// Using private fork of controller-tools. See commit msg for more context
 	// as to why we are using a private fork.
-	sigs.k8s.io/controller-tools => github.com/christarazi/controller-tools v0.3.1-0.20200803222741-1b051103a137
+	sigs.k8s.io/controller-tools => github.com/christarazi/controller-tools v0.3.1-0.20200814033407-d944debcff34
```

Comment:
:100: 

Just curious if we have any plan to move this to cilium org (similar to client-go), this will help to avoid similar non-existent hash issue to some capacity :).

Reply:
Yes, I think in the near-future we should move this repo under the Cilium org. Once we merge in https://github.com/cilium/cilium/pull/11477 and finish all the followups (https://github.com/cilium/cilium/issues/12737), likely the controller-tools fork will stabilize and we can "promote" it to be under the Cilium org.